app/javascript/article-form/components/EditorBody.jsx

```diff
@@ -50,9 +52,17 @@ export const EditorBody = ({
     onDragExit,
   });
 
+  const setPasteElement = usePasteImage({
+    onPaste: handleImagePasted(
+      handleImageSuccess(textAreaRef),
+      handleImageFailure,
+    )
+  }).setElement;
```

Comment:
Refactor (blocking): Ties into my refactor below.

```suggestion
  const setPasteElement = usePasteImage({
    onPaste: handleImagePasted(
      handleImageSuccess(textAreaRef),
      handleImageFailure,
    )
  });
```